### ISSUE TITLE
Promote the open positions for the KCMO Health Dept. Contact Tracing positions

### ISSUE THREAD

**Issue by JakeLaCombe** [codeforkansascity/kc-covid-response#6]
The KCMO Health Dept. is looking to hire close to 100 disease detectives to assist with the COVID outbreak. Training is also available at https://www.coursera.org/learn/covid-19-contact-tracing, which can introduce you to what it means to contact trace. Perhaps we could have it in like a getting involved link and group it with other help? I think it would be a good service to the city to help promote these positions.

**Comment by aarondeacon** [comment]
Yes, this is a great idea. There will be advertisements for contact tracing positions for several months. We can definitely include a list of job postings...should be some decent way to visualize that
